### ISSUE TITLE
NotImplementedError: The list_tags_for_resource action has not been implemented

### ISSUE THREAD

**Issue by The-TJ** [spulec/moto#3226]
I'm trying to list tags for a resource in an organization but the list_tags_for_resource method shows up as not implemented. Looking at the Organizations model, I can see it is already implemented. I also get the same error when trying to use the tag_resource method.
```
self = <moto.organizations.responses.OrganizationsResponse object at 0x7f30d10067d0>
[CPython37-test:stdout]
[CPython37-test:stdout]     def call_action(self):
[CPython37-test:stdout]         headers = self.response_headers
[CPython37-test:stdout]         action = camelcase_to_underscores(self._get_action())
[CPython37-test:stdout]         method_names = method_names_from_class(self.__class__)
[CPython37-test:stdout]         if action in method_names:
[CPython37-test:stdout]             method = getattr(self, action)
[CPython37-test:stdout]             try:
[CPython37-test:stdout]                 response = method()
[CPython37-test:stdout]             except HTTPException as http_error:
[CPython37-test:stdout]                 response = http_error.description, dict(status=http_error.code)
[CPython37-test:stdout]
[CPython37-test:stdout]             if isinstance(response, six.string_types):
[CPython37-test:stdout]                 return 200, headers, response
[CPython37-test:stdout]             else:
[CPython37-test:stdout]                 if len(response) == 2:
[CPython37-test:stdout]                     body, new_headers = response
[CPython37-test:stdout]                 else:
[CPython37-test:stdout]                     status, new_headers, body = response
[CPython37-test:stdout]                 status = new_headers.get('status', 200)
[CPython37-test:stdout]                 headers.update(new_headers)
[CPython37-test:stdout]                 # Cast status to string
[CPython37-test:stderr] Fully configured from brazil-path's testbuild
[CPython37-test:stderr] Pytest failed with exit code: 1 [TESTS FAILED]
[CPython37-test:stdout]                 if "status" in headers:
[CPython37-test:stdout]                     headers['status'] = str(headers['status'])
[CPython37-test:stdout]                 return status, headers, body
[CPython37-test:stdout]
[CPython37-test:stdout]         if not action:
[CPython37-test:stdout]             return 404, headers, ''
[CPython37-test:stdout]
[CPython37-test:stdout]         raise NotImplementedError(
[CPython37-test:stdout] >           "The {0} action has not been implemented".format(action))
[CPython37-test:stdout] E       NotImplementedError: The list_tags_for_resource action has not been implemented
```

**Comment by bblommers** [comment]
Hi @The-TJ, thanks for raising this.
What version are you on? This method should be available from moto >= 1.3.15.dev122

**Comment by bblommers** [comment]
Going to close this due to a lack of response, but feel free to let me know if this is still an issue.
